3_visualize/src/svg_utils_vue.R

```diff
@@ -5,8 +5,14 @@ gwl_to_peak <- function(file_out, gw_anomaly_data_w_colors){
            path_y = round(50-daily_quant, digits = 0)) %>%
     #filter(!is.na(quant)) %>%
     mutate(site_no = paste0('gwl_', site_no)) %>%
-    select(site_no, day_seq, daily_quant, path_y) %>%
+    select(site_no, day_seq, daily_quant, path_y) %>% 
+    
+    # Convert to `data.table` for fast reshaping, then back 
+    # to tibble. Note that `reshape2::dcast()` is deprecated.
+    setDT() %>% 
```

Comment:
☹️  `dcast` has been one of my most used functions, I'm resistant to the change

Reply:
Can still use `dcast()` but has to be from the `data.table` package not `rehsape2`. The warning I got said the next update would turn it into an error!